liquid_node/configuration.py

```diff
@@ -159,6 +147,25 @@ def load_job(self, name, job_config):
     def app_url(self, name):
         return f'{self.liquid_http_protocol}://{name}.{self.liquid_domain}'
 
+    def _jobs(self):
+        all_jobs = []
+        jobs_to_start = []
+
+        for core_app in Configuration._core_apps:
+            if core_app in Configuration._allways_on_apps and self.ini.has_option('apps', core_app):
+                raise RuntimeError(f'The app "{core_app}" is always on.')
+
+            app_jobs = list(import_string(f'liquid_node.jobs.{core_app}.jobs'))
+            all_jobs.extend(app_jobs)
+            if self.is_app_enabled(core_app):
+                jobs_to_start.extend(app_jobs)
+
+        return all_jobs, jobs_to_start
```

Comment:
Since we shouldn't stop jobs when running deploy, you can remove `all_jobs` from here and only return the `jobs_to_start` list.